test/codeu/chat/server/DeleteConversationTest.java

```diff
@@ -84,4 +85,36 @@ public void testdeleteMultipleConversations() {
 	  assertTrue(model.conversationById().first(testConversation1.id) == null&& model.conversationById().first(testConversation2.id)==null);
   }
   
+  @Test
+  public void testdeleteConversationDoesNotExistEmpty() {
+	  
+	  controller.deleteConversation("conversation");
+	  
+	  assertNull(model.conversationByText().first("conversation"));
+  }
+  
+  @Test
+  public void testdeleteConversationDoesNotExistNonEmpty() {
+	  final User user = controller.newUser("user");
+	  final Conversation conversation = controller.newConversation("conversation", user.id);
+	  
+	  controller.deleteConversation("convo");
```

Comment:
An error message is output alerting the client that me does not exist, this was handled from the client side so it was manually tested.